dev-tools/cherrypick_pr

```diff
@@ -106,33 +108,32 @@ def main():
     else:
         token = open(expanduser("~/.github_token"), "r").read().strip()
         base = "https://api.github.com/repos/elastic/beats"
-        s = requests.Session()
-        s.headers.update({"Authorization": "token " + token})
+        session = requests.Session()
+        session.headers.update({"Authorization": "token " + token})
 
-        original_pr = s.get(base+"/pulls/"+args.pr_number).json()
+        original_pr = session.get(base + "/pulls/" + args.pr_number).json()
 
         # get the github username from the remote where we pushed
         remote_url = check_output("git remote get-url {}".format(remote),
                                   shell=True)
         remote_user = re.search("github.com:(.+)/beats", remote_url).group(1)
 
         # create PR
-        r = s.post(base+"/pulls", json=dict(
-            title="Cherry-pick to {}: {}"
-                  .format(args.to_branch, original_pr["title"]),
+        request = session.post(base + "/pulls", json=dict(
+            title="Cherry-pick to {}: {}".format(args.to_branch, original_pr["title"]),
             head=remote_user + ":" + tmp_branch,
             base=args.to_branch,
             body="Cherry-pick of PR #{} to {} branch. Original message: \n\n{}"
-                 .format(args.pr_number, args.to_branch, original_pr["body"])
+            .format(args.pr_number, args.to_branch, original_pr["body"])
         ))
-        if r.status_code > 299:
-            print("Creating PR failed: {}".format(r.json()))
+        if request.status_code > 299:
+            print("Creating PR failed: {}".format(request.json()))
             sys.exit(1)
-        new_pr = r.json()
+        new_pr = request.json()
 
         # add labels
-        s.post(base+"/issues/{}/labels".format(new_pr["number"]),
-               json=["backport", "review"])
+        session.post(
+            base + "/issues/{}/labels".format(new_pr["number"]), json=["backport", "review"])
```

Comment:
We probably have different settings in the editor, because this version yields an pep error here for me (`line too long (93 > 79 characters)`). I _think_ I am just using the default pep settings.

We didn't yet impose pep/flake in the CI so far because they can be annoying sometimes and the python code that we have is generally not mission critical. Perhaps it's worth discussing this again.